### DISCUSSION TITLE
How to set password when using redis cluster's <nodes.conf> file

### DISCUSSION THREAD

**Discussion by tsabados** [leonchen83/redis-cli-tool#34] [General]
How to set the password when using Redis cluster's <nodes.conf> file?
For example, when using this command:
```
rst -s redis://127.0.0.1:6379 -c ./nodes.conf -r
```
I get this output
```
failed to connect cluster nodes, reason : NOAUTH Authentication required.
failed to connect cluster nodes, reason : NOAUTH Authentication required.
failed to connect cluster nodes, reason : NOAUTH Authentication required.
```

**Comment by leonchen83** [comment]
@TheZbunj 
Hi, if use `nodes.conf` file, can't set cluster password, but you can use cluster host and port(any of them) directly to migrate data from single redis to cluster. for example
```
rst -s redis://127.0.0.1:6379 -m redis://clusterhost:clusterport?authPassword=yourclusterpassword -d 0 -r
```

**Comment by leonchen83** [comment]
and be notice that the difference between `rst` and `rmt`. most of scenario we use `rmt` and not `rst` see [difference-between-rmt-and-rst](https://github.com/leonchen83/redis-rdb-cli#difference-between-rmt-and-rst)

**Comment by tsabados** [comment]
Thank you! 

I ended up using `rmt` in a loop to migrate the remote cluster to local.
